lib/Runtime/Library/JavascriptExternalFunction.cpp

```diff
@@ -424,9 +430,15 @@ namespace Js
             TTD::TTDNestingDepthAutoAdjuster logPopper(scriptContext->GetThreadContext());
             TTD::NSLogEvents::EventLogEntry* callEvent = elog->RecordExternalCallEvent(externalFunction, scriptContext->GetThreadContext()->TTDRootNestingCount, args.Info.Count, args.Values, true);
 
+            StdCallJavascriptMethodInfo info = {
+                callInfo.Count > 0 ? args[0] : scriptContext->GetLibrary()->GetNull(),
```

Comment:
Just a comment to note that the above conversation should also be updated here (re: null or undefined, callInfo.Count > 0)